### ISSUE TITLE
`blowfish_hash` doesn't work

### ISSUE THREAD

**Issue by delirious-lettuce** [Ekultek/Dagon#25]
# Issue

`lib/algorithms/hashing_algs.blowfish_hash` doesn't work

# Any idea what caused this to happen?
I was reading through the [`passlib`](https://passlib.readthedocs.io/en/stable/lib/passlib.hash.bcrypt.html#) docs when I noticed this line:

> Note It is strongly recommended that you install bcrypt when using this hash.

I checked the `requirements.txt` and `bcrypt` was not in it. 

For more information on the backends used by `passlib`:

https://passlib.readthedocs.io/en/stable/lib/passlib.hash.bcrypt.html#index-0

  - Exact commands used:
```python
# added underneath the imports of `hashing_algs.py`
print bcrypt.hash('test_string')

```

# Exact error message:
```python
MissingBackendError: bcrypt: no backends available -- recommend you install one (e.g. 'pip install bcrypt')

```

**Comment by Ekultek** [comment]
I guess I'll just add `bcrypt` to `requirments.txt`that should fix the problem I believe.

**Comment by Ekultek** [comment]
fixed via 07cb584

**Comment by Ekultek** [comment]
check out #26
